src/scalars/iso-date/Date.ts

```diff
@@ -71,7 +71,7 @@ export const GraphQLDateConfig: GraphQLScalarTypeConfig<Date, string> =
       );
     },
     extensions: {
-      codegenScalarType: 'Date | string',
```

Comment:
For client, you receive `string`, not `Date.
So we need to satisfy both client and servers :/

Reply:
Ahhhhhh, hot dang, I see 😭 Thanks for letting me know, I'll close this all out